browser-interface/packages/shared/session/sagas.ts

```diff
@@ -321,7 +321,9 @@ async function createAuthIdentity(requestManager: RequestManager, isGuest: boole
 
   const ssoIdentity = await SingleSignOn.getIdentity(address)
 
-  if (!ssoIdentity) {
+  const validIdentity = !ssoIdentity ? false : isSessionExpired({ identity: ssoIdentity })
```

Comment:
Maybe I'm wrong, but this logic would mean:
- If no identity is obtained, it is NOT valid
- If the identity is expired, validIdentity would be TRUE
The last one should be the other way around right?

Reply:
You're right. I have just fixed it, the logic was wrong